sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs

```diff
@@ -336,7 +336,7 @@ public partial class AfdSecretData : Azure.ResourceManager.Models.ResourceData
     {
         public AfdSecretData() { }
         public Azure.ResourceManager.Cdn.Models.AfdDeploymentStatus? DeploymentStatus { get { throw null; } }
-        public Azure.ResourceManager.Cdn.Models.SecretDefinition Parameters { get { throw null; } set { } }
+        public Azure.ResourceManager.Cdn.Models.SecretProperties Parameters { get { throw null; } set { } }
```

Comment:
Should we also rename the property name to `Properties` to align the name of its type?

Reply:
Nice suggestion! I will update.